conf/vlan_filters.conf.example

```diff
@@ -113,11 +112,10 @@
 #     owner.can_sponsor
 #     owner.unregdate
 #     owner.category
-#     radius_request
+#     radius_request.RADIUS_request_attributes (see below)
 #
-# Attribute can be:
-# - for the radius_request
-#     All the attributes you can have in the RADIUS request (run FreeRADIUS in debug mode to see these attributes)
+# For the radius_request filter, you can use all the attributes you can have
+# in the RADIUS request (run FreeRADIUS in debug mode to see these attributes)
```

Comment:
we could also talk about looking in the RADIUS tab of an audit log detailed entry